src/Pos/Wallet/Web/ClientTypes.hs

```diff
@@ -189,24 +253,72 @@ instance Default CWalletMeta where
 -- | Client Wallet (CW)
 -- (Flow type: walletType)
 data CWallet = CWallet
-    { cwAddress :: !CAddress
-    , cwAmount  :: !Coin
-    , cwMeta    :: !CWalletMeta
+    { cwAddress  :: !CWalletAddress
+    , cwMeta     :: !CWalletMeta
+    , cwAccounts :: ![CAccount]
     } deriving (Show, Generic, Typeable)
 
 -- | Query data for wallet creation
--- (wallet meta + backup phrase)
 data CWalletInit = CWalletInit
-    { cwBackupPhrase :: !BackupPhrase
-    , cwInitMeta     :: !CWalletMeta
+    { cwInitMeta   :: !CWalletMeta
+    , cwInitWSetId :: !(CAddress WS)
     } deriving (Show, Generic)
 
 -- | Query data for redeem
 data CWalletRedeem = CWalletRedeem
-    { crWalletId :: !CAddress
+    { crWalletId :: !CWalletAddress
     , crSeed     :: !Text -- TODO: newtype!
     } deriving (Show, Generic)
 
+-- | Meta data of 'CWalletSet'
+data CWalletSetMeta = CWalletSetMeta
+    { cwsName         :: !Text
+    , cwsBackupPhrase :: !BackupPhrase
+    } deriving (Show, Eq, Generic)
+
+instance Default BackupPhrase where
+    def = mkBackupPhrase
```

Comment:
Why not? If not `def`, it would be called `someBackPharse` (since it's used often in doc examples)